.travis.yml

```diff
@@ -19,19 +19,23 @@ script:
   - ./.ci/local-registry.sh
     # Update deployments to reference the local registry
   - ./.ci/replace-refs.sh
+    # Install kafka via Strimzi
+  - ./start-strimzi.sh
+    # Wait for strimzi custom resource to be ready (this will be ready once the
+    # zookeeper and kafka cluster have been created and are ready)
+  - export OPERATOR=`kubectl get pod -l name=strimzi-cluster-operator -o jsonpath="{.items[0].metadata.name}" -n kafka` && echo $OPERATOR
+  - kubectl wait --for=condition=Ready kafkas/kafka-cluster -n kafka --timeout=180s || kubectl logs $OPERATOR -n kafka
```

Comment:
I think you could do `kubectl logs deployment/strimzi-cluster-operator -n kafka` and avoid looking up the pod

Reply:
Good idea!